src/openrct2/paint/Paint.cpp

```diff
@@ -891,9 +890,9 @@ paint_struct* sub_98197C(
 
     session->LastRootPS = ps;
 
-    LocationXY16 attach = { (int16_t)ps->bounds.x, (int16_t)ps->bounds.y };
+    CoordsXY attach = { ps->bounds.x, ps->bounds.y };
 
-    rotate_map_coordinates(&attach.x, &attach.y, session->CurrentRotation);
+    attach = attach.Rotate(session->CurrentRotation);
```

Comment:
You could just do `CoordsXY attach = CoordsXY{ ps->bounds.x, ps->bounds.y }.Rotate(session->CurrentRotation);`
Not sure what is better readability wise.

Reply:
I think this line is small enough for this to be ok, so I'm following your suggestion.